drools-osgi/drools-karaf-itests-kjar/pom.xml

```diff
@@ -18,4 +18,17 @@
     </dependency>
   </dependencies>
 
+  <packaging>kjar</packaging>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-maven-plugin</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
```

Comment:
Actually, looking at this again, I am afraid this won't work, because plugin will be in the same reactor build as this module. We had to exclude the kie-maven-plugin-example from the build for the same reason.